Godeps/Godeps.json

```diff
@@ -11,7 +11,7 @@
 	"Deps": [
 		{
 			"ImportPath": "github.com/Clever/pathio",
-			"Comment": "v3.0.0-2-gc9ab5fa",
+			"Comment": "v2.0.0-23-gc9ab5fa",
```

Comment:
huh- so we had to downgrade a major version of pathio - will this cause a problem? what changed in Pathio?